http/http.go

```diff
@@ -113,7 +113,7 @@ func (s *Server) newResponse(r *http.Request) (Response, error) {
 func (s *Server) newPortResponse(r *http.Request) (PortResponse, error) {
 	lastElement := filepath.Base(r.URL.Path)
 	port, err := strconv.ParseUint(lastElement, 10, 16)
-	if err != nil || port < 1 || port > 65355 {
+	if err != nil || port < 1 || port > 65535 {
 		return PortResponse{Port: port}, fmt.Errorf("invalid port: %d", port)
```

Comment:
Can you change the format string to `%s` here and pass `lastElement` instead? That should fix the error message.